.circleci/config.yml

```diff
@@ -1,27 +1,66 @@
-version: 2
+version: 2.1
+commands:
+  checkout-test:
+    steps:
+      - run:
+          name: Checkout tests
+          command: cd .. && git clone -b ${TESTS_BRANCH} --single-branch git@github.com:twilio/selenoid-container-tests.git
```

Comment:
`cd ..` would it take the current directory beyond root directory?

Reply:
No, it won't. Each step is run in the default working directory ~/project (where the code was checked out). Therefore the tests will be checked out one level above selenoid-images code, as needed, in ~